### PR TITLE
Added listing display challenge to README file

### PR DESCRIPTION
**Pull request by clinturbin**
For this branch:
  - In README.md, I added the challenge for moving the listing for a selected marker to the top of the display

@running-on-sunshine: Could you please review?

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -42,4 +42,28 @@ This is a frontend group project built with HTML/CSS, JavaScript, and the Google
...
+
+* MOVE LISTING FOR SELECTED MARKER TO TOP OF DISPLAY  
+    
+  - How the feature works:  
+      1) You select a marker on the map
+      2) An Info Window for that marker is opened
+      3) The listing for the selected marker is moved to the top of the display
+  
+  - Problem:  
+    When someone selects a marker on the map, the info window opening would sometimes cause the map viewport to change. The viewport change would then cause the listings display to refresh. This would then overwrite the call to display the selected marker first. So, in the end you would just see an unordered display of listings.
```
Comment (running-on-sunshine) [inline README.MD]: Thoughts: 
- I really liked how you tried to break down the problem into bite-sized pieces! 🙂
- Possible suggestion! Maybe something like the following below? Mostly replacing 'someone' and 'you' with 'user.'

```When the user selects a marker on the map, the marker’s informational window opening would sometimes cause the viewport to shift. This viewport shift would then trigger the listings results to refresh, and then overwrite the initial call to display the user’s selected marker first. Thus, the user would continue to see an unordered display of listings.```

File: README.MD [hunk 2]
```diff
@@ -42,4 +42,28 @@ This is a frontend group project built with HTML/CSS, JavaScript, and the Google
...
-  Save the file and you are ready to run the application. Yay! :)
\ No newline at end of file
+  Save the file and you are ready to run the application. Yay! :)
+
+## Challenges
+
+* MOVE LISTING FOR SELECTED MARKER TO TOP OF DISPLAY  
+    
+  - How the feature works:  
+      1) You select a marker on the map
```
Comment (running-on-sunshine) [inline README.MD]: Thoughts:
- I really like that you added a little section explaining how the feature works! 🙂
- Possible suggestion below. Maybe replacing 'you' with 'user'? 

```
The user selects a marker on the map
```

File: README.MD [hunk 3]
```diff
@@ -42,4 +42,28 @@ This is a frontend group project built with HTML/CSS, JavaScript, and the Google
...
+      2) An Info Window for that marker is opened
+      3) The listing for the selected marker is moved to the top of the display
+  
+  - Problem:  
+    When someone selects a marker on the map, the info window opening would sometimes cause the map viewport to change. The viewport change would then cause the listings display to refresh. This would then overwrite the call to display the selected marker first. So, in the end you would just see an unordered display of listings.
+  
+  - Solution:  
+    To fix this problem, the displaySelectedMarkerFirst function call was wrapped in a setTimeout function.  This gave the map a half second to refresh itself before updating the display listings.
+    ```
```
Comment (running-on-sunshine) [inline README.MD]: This is a really great work, @clinturbin! 🙂

Thoughts: 
- The solution and explanation for it is straight to the point and easy to understand! 
- One minor thing that could be resolved. There seems to be an extra space or two before the second sentence starting with "This gave the map..."

**Comment by clinturbin** [comment]
Just made changes to the README file.  Thank You for the suggestions!!

**Review by running-on-sunshine — APPROVED** [review]
Thank you so much for making the suggested changes!! 🙂

Feel free to merge to master when you are ready! 👍

**Pull request merged**